backend/protos/mgmt/v1alpha1/user_account.proto

```diff
@@ -172,10 +172,7 @@ message SetAccountOnboardingConfigResponse {
 }
 
 message AccountOnboardingConfig {
-  bool has_created_source_connection = 1;
-  bool has_created_destination_connection = 2;
-  bool has_created_job = 3;
-  bool has_invited_members = 4;
```

Comment:
This is a breaking change and properties in proto shouldnt ever be renamed or re-used.

So what you actually want to do here is to instead deprecate these by marking them as deprecated (you can see in other protos uses of the deprecated comment.

Then the `has_completed_onboarding` should be a new property that starts at `5`;

Reply:
From the outside these can all just default to false and everything is gucci since the frontend will only care about the new property after this.

Reply:
You will also need to update the user service integration tests to accommodate your new changes.